### PR TITLE
corrscope: 0.8.1 -> 0.9.0

### LINKED ISSUE #295349: Update request: corrscope 0.8.1 → 0.9.0
- Package name: corrscope
- Latest released version: 0.9.0

- Current version on the unstable channel: 0.8.1
- Current version on the stable/release channel:0.8.1

- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

@OPNA2608 

My apologies for accidentally publishing before i was done with the issue.

-----

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by OPNA2608**
## Description of changes

Closes #295349

CC @FavoritoHJS

Only tested that it launches rn, can only do proper testing in afew hours.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/co/corrscope/package.nix [hunk 1]
```diff
@@ -1,49 +1,39 @@
...
 , testers
 , corrscope
 }:
 
-mkDerivationWith python3Packages.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "corrscope";
-  version = "0.8.1";
+  version = "0.9.0";
   format = "pyproject";
```
Comment (SuperSandro2000) [inline pkgs/by-name/co/corrscope/package.nix:13]:
```suggestion
  pyproject = true;
```

File: pkgs/by-name/co/corrscope/package.nix [hunk 2]
```diff
@@ -1,49 +1,39 @@
...
     wrapQtAppsHook
-  ] ++ (with python3Packages; [
+  ]) ++ (with python3Packages; [
     poetry-core
     pythonRelaxDepsHook
   ]);
 
   buildInputs = [
-    ffmpeg
+    ffmpeg-full
```
Comment (SuperSandro2000) [inline pkgs/by-name/co/corrscope/package.nix:32]: Shouldn't only ffmpeg be used and not full? What are we missing from the normal variant?

Comment (OPNA2608) [reply pkgs/by-name/co/corrscope/package.nix:32]: https://github.com/NixOS/nixpkgs/issues/271863#issuecomment-1837484140

> `corrscope`: Uses `ffplay` for export preview, which depends on `withFullDeps`.

File: pkgs/by-name/co/corrscope/package.nix (hunk 1 above)
Comment (OPNA2608) [reply pkgs/by-name/co/corrscope/package.nix]: Applied.

**Comment by OPNA2608** [comment]
@OfBorg eval

File: pkgs/by-name/co/corrscope/package.nix (hunk 2 above)
Comment (SuperSandro2000) [reply pkgs/by-name/co/corrscope/package.nix:33]: see #296128

Comment (OPNA2608) [reply pkgs/by-name/co/corrscope/package.nix]: Switched to regular `ffmpeg` now, and tested that it works fine.

**Pull request merged**